context/app/static/js/components/detailPage/entityHeader/EntityHeader/EntityHeader.tsx

```diff
@@ -41,11 +42,7 @@ function Header() {
 
   return (
     <AnimatedPaper style={styles} elevation={4} data-testid="entity-header">
-      <EntityHeaderContent
-        assayMetadata={assayMetadata}
-        shouldDisplayHeader={shouldDisplayHeader}
-        vizIsFullscreen={vizIsFullscreen}
-      />
+      <EntityHeaderContent assayMetadata={assayMetadata} shouldDisplayHeader={shouldDisplayHeader} vizIsFullscreen />
```

Comment:
If `vizIsFullscreen` is always set to true, the header always displays controls for a potentially non-existent vitessce visualization:
![image](https://github.com/user-attachments/assets/51a6a917-161e-4793-9439-bea52dfc4def)


```suggestion
      <EntityHeaderContent assayMetadata={assayMetadata} shouldDisplayHeader={shouldDisplayHeader} vizIsFullscreen={vizIsFullscreen} />
```

With this suggestion:

![image](https://github.com/user-attachments/assets/5da9efd6-940b-46a7-b639-57f6002ddadc)

Reply:
Ah good catch! That was an unintentional change from trying to resolve eslint errors.